cro-clib/src/jsonrpc.rs

```diff
@@ -69,7 +69,7 @@ pub unsafe extern "C" fn cro_jsonrpc_call(
     request: *const c_char,
     buf: *mut c_char,
     buf_size: usize,
-    progress_callback: Option<ProgressCallback>,
+    progress_callback: ProgressCallback,
```

Comment:
The `Option` is to handle no callback scenario.

Reply:
i see, but Option<..> cannot be used in c-language.(gcc error)
it will be done by giving NULL as argument
i'll add code for that condition

Reply:
exported callback function type, example can use it to give function pointer correctly.

Reply:
typedef int32_t (*ProgressCallback)(uint64_t, uint64_t, uint64_t, const void*);
typedef ProgressCallback Option_ProgressCallback;